CSP_Minizinc_Code.py

```diff
@@ -75,18 +79,18 @@ def update_repository(self):
 class Safety_Monitor_Selector(Selector):
     def __init__(self,repo_image):
         #Initially the attributes are set to null and no safety monitor is selected by default.
-        self.__current_context = (False,False,False)
+        self.__current_context = (False,False)

```

Comment:
Its better to add a comment what current context contains (.i.e. Robot in motion and gripper status)